client/src/components/RichTextEditor.js

```diff
@@ -64,11 +68,71 @@ const RichTextEditor = ({
       editor.children = createSlateValue(value)
       editor.onChange()
     }
-  }, [editor, previousValue, readOnly, value])
+  }, [editor, previousValue, readOnly, disableFullSize, value])
 
   const renderElement = useCallback(props => <Element {...props} />, [])
   const renderLeaf = useCallback(props => <Leaf {...props} />, [])
 
+  const editableRef = useRef()
+  const toolbarRef = useRef()
+
+  useEffect(() => {
+    editableRef.current = ReactEditor.toDOMNode(editor, editor)
+  }, [editor])
+
+  const handleFullSizeMode = isFullSize => {
+    handleBannerHeight()
+    handleToolbarHeight()
+    setShowFullSize(isFullSize)
+  }
+  const handleBannerHeight = () => {
+    const topOffset =
+      document.getElementById("securityBannerContainer").getBoundingClientRect()
+        .bottom + "px"
```

Comment:
I wonder if this is the best way to get the height of the `securityBannerContainer`.

Can we use the value directly from the component, rather then this low level calls?:
https://github.com/NCI-Agency/anet/blob/c847af86f9f4bb27bb8564c82ecafee2bc116efb/client/src/components/SecurityBanner.js#L46